src/utils/utils/utils.ts

```diff
@@ -32,6 +36,9 @@ export const get = (obj: unknown, path: string | string[], defaultValue = undefi
 
 export const isUpstream = (window as any).SERVER_FLAGS?.branding === 'okd';
 
+export const getHyperconvergedNamespace = () =>
```

Comment:
this name isnt right, this func differ between cnv to hyperconverge , maybe change it to getDefaulktOperatonNS or something else?